tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/Summary.java

```diff
@@ -248,6 +258,25 @@ private static boolean autoSnapshotsAreEnabled() {
         }
     }
 
+    private static Calendar getNextSnap() {
+        final Calendar when = Calendar.getInstance(TimeZone.getTimeZone(AUTO_SNAP_TIME_ZONE));
+        when.setLenient(true); // December 32 = January 1
+        final int tomorrow = when.get(Calendar.DAY_OF_YEAR) + 1;
+        when.set(Calendar.DAY_OF_YEAR, tomorrow);
+        when.set(Calendar.HOUR_OF_DAY, AUTO_SNAP_HOUR_OF_DAY);
+        when.set(Calendar.MINUTE, 0);
+        when.set(Calendar.SECOND, 0);
+        when.set(Calendar.MILLISECOND, 0);
+        return when;
+    }
```

Comment:
If the server restarts between midnight and 1 am (AUTO_SNAP_HOUR_OF_DAY), there won't be a snapshot until the next day -- I'll address that in my next PR